torch/_refs/__init__.py

```diff
@@ -3275,6 +3275,20 @@ def new_empty_strided(
     )
 
 
+@register_decomposition(torch.ops.aten.zeros)
+def zeros(
```

Comment:
What about the `out` parameter?

Reply:
This is the decomp for functional version of zero.
For out-variant ops, I think we need a separated decomp func.

Reply:
Can't we just use the `out` wrapper? It should register the correct decomposition as well.

Reply:
ic what you mean... added.